package.json

```diff
@@ -31,20 +31,20 @@
     "react-collapse": "Trengs for datovelgeren, overrider versjonen som ligger der"
   },
   "dependencies": {
-    "@navikt/ds-css": "^1.5.4",
-    "@navikt/ds-icons": "^1.5.4",
-    "@navikt/ds-react": "^1.5.4",
-    "@navikt/ds-react-internal": "^1.5.4",
-    "@navikt/ds-tokens": "^1.5.4",
-    "@navikt/familie-backend": "9.0.0",
-    "@navikt/familie-dokumentliste": "^4.0.5",
-    "@navikt/familie-endringslogg": "^3.0.9",
-    "@navikt/familie-form-elements": "^6.3.7",
-    "@navikt/familie-header": "^6.0.5",
-    "@navikt/familie-ikoner": "^4.0.7",
-    "@navikt/familie-logging": "^3.0.5",
-    "@navikt/familie-typer": "^5.0.4",
-    "@navikt/familie-visittkort": "^4.0.9",
+    "@navikt/ds-css": "2.0.x",
+    "@navikt/ds-icons": "2.0.x",
+    "@navikt/ds-react": "2.0.x",
+    "@navikt/ds-react-internal": "2.0.x",
+    "@navikt/ds-tokens": "2.0.x",
```

Comment:
Endret til å bruke `x` i stedet for `^` her  -> `2.0.x` i stedet for `^2.0.0`
Litt for å ha mer kontrollerte oppdateringer av minor-versjoner  fra ds.

Reply:
Eks at man kjører yarn lokalt og då automatiskt tar i bruk en ny minor versjon uten å tenke på det 👀

Reply:
Hva tenker dere, høres det greit ut?

Reply:
I utgangspunktet enig, men noen grunn til å ikke bare kreve `2.0.0`?

Reply:
@eirarset `^2.0.0` installerer nye minors, dvs `2.9.0` hvis den finnes, endringen installerer kun patches, dvs `2.0.9`

Reply:
Ja, skjønner. Jeg tenkte bare på om det var lurt om vi også "låser" patch-versjon, så vi evt. slipper å sitte med en versjon lokalt i og en annen i dev/prod. (Dette var jo også et problem før). 

Ingen sterke meninger om dette, tenkte bare på om det kunne være lurt :) Og endringen din er uansett en forbedring 👍